.travis.yml

```diff
@@ -1,4 +1,4 @@
 language: node_js
 
 node_js:
- - stable
+ - lts/argon
```

Comment:
I think you can add multiple entries here right? So let's do

 - lts/argon
 - stable

so that it runs on both.

Reply:
One way to find out...